setup.py

```diff
@@ -170,7 +170,7 @@ def str_from_file(name):
           ],
       },
       classifiers=[
-          "Topic :: System :: Benchmark",
+          "Topic :: System :: OpenSearch-Benchmark",
```

Comment:
Seems like this was still `Benchmark` but have updated it to `OpenSearch-Benchmark`